pipelines/rj_iplanrio__test/flow.py

```
@@ -3,16 +3,27 @@
+
 
 
 @flow(log_prints=True)
 def rj_iplanrio__test(table_id):
     rename_current_flow_run_task(new_name=table_id)
-    for x in range(10):
-        time.sleep(1)
-        print(f"{x} - Hello World")
+    for x in range(5):
+        time.sleep(2)
+        logger.debug(f"Total, %i %s", x, "DEBUG")
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)